### ISSUE TITLE
Instructions for setting up DNS are not clear on how to find the appropriate DNS server

### ISSUE THREAD

**Issue by bpm-ms** [Microsoft/service-fabric#39]
When trying to figure out what DNS server to use to enable Docker internet connectivity behind a strict firewall, it can be unclear or confusing on where to find your corporate DNS server.  It also could use additional clarification that the default DNS server is google (8.8.8.8), and if you are having internet problems, try to ping that server first, and if it doesn't work THEN update your daemon.json with the appropriate DNS server.  I think any DNS server you can ping should work.

**Comment by vturecek** [comment]
@bpm-ms is this specific to the build container, or containers in general? Just want to make sure it doesn't get confused with Service Fabric's DNS Service.

**Comment by bpm-ms** [comment]
This is true for containers in general.  However, it's a common issue that people face (just like how to install docker, and how to run docker without sudo)

**Comment by anmolah** [comment]
@bpm-ms is the work here a documentation update? What work is this issue tracking?

**Comment by bpm-ms** [comment]
Yes, documentation update. Just need some clarification or instructions on how you might figure out what your DNS servers IP is in the section about connecting behind a corporate firewall
